jobs/sentry-error-loader/sentry_error_loader.py

```diff
@@ -38,20 +39,32 @@ def fetch_and_clean_from_clickhouse(project_slug, target_date):
     """
 
     print(f"Gathering Sentry event data for project {project_slug}")
-    client = clickhouse_connect.get_client(host="localhost", port=8123)
+
+    extract = SentryExtract(
+        project_slug=project_slug,
+        dt=target_date,
+        execution_ts=pendulum.now(),
+        filename="events.jsonl.gz",
+    )
+
+    client = clickhouse_connect.get_client(
+        host="sentry-clickhouse.sentry.svc.cluster.local", port=8123
```

Comment:
nit: ideally these should be defaults and overrideable via env vars so we don't have to publish a new image to change the host/port